### PR TITLE
Work frbr fields

### PR DESCRIPTION
**Pull request by longhotsummer**
Store FRBR URI fields explicitly on the work. This makes reporting and analysis from the db simpler.

### CODE REVIEW

File: indigo_api/migrations/0015_work_frbr_fields.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        migrations.AlterField(
+            model_name='task',
+            name='extra_data',
+            field=models.JSONField(blank=True, null=True),
+        ),
+        migrations.AlterField(
+            model_name='work',
+            name='properties',
+            field=models.JSONField(blank=True, default=dict),
+        ),
```
Comment (goose-life) [inline indigo_api/migrations/0015_work_frbr_fields.py:57]: Are these alterations part of this change? or just updates to allow None?

**Review by goose-life — APPROVED** [review]
I'm not sure I get the significance of moving the properties off the DocumentMixin and onto the Document; or why the `work_uri`'s properties are being used there rather than the `work`'s.

but lgtm!

File: indigo_api/migrations/0015_work_frbr_fields.py (hunk 1 above)
Comment (longhotsummer) [reply indigo_api/migrations/0015_work_frbr_fields.py:57]: These are old ones only showing up now.

**Comment by longhotsummer** [comment]
> I'm not sure I get the significance of moving the properties off the DocumentMixin and onto the Document; or why the `work_uri`'s properties are being used there rather than the `work`'s.
> 
> but lgtm!

Otherwise they clash with signoff. They do look a bit odd in this PR :)

**Pull request merged**